etc/openqa/openqa.ini

```diff
@@ -121,6 +121,12 @@
 ## - when set to "best-effort" openQA jobs are started even if the update failed
 ## - when set to "strict" openQA jobs will be blocked until the update succeeded or set to incomplete when the retries for updating are exhausted
 #git_auto_update_method = best-effort
+# whether openQA should attempt to display needles according to the original commits in the web UI
```

Comment:
Please use the consistent comment style as above. Also please state the consequence or alternatives. Right now the sentence can be read as something that one always wants. But what is the drawback?